src/main/scala/com/microsoft/hyperspace/actions/RefreshAction.scala

```diff
@@ -37,9 +37,9 @@ class RefreshAction(
     extends RefreshActionBase(spark, logManager, dataManager) {
 
   final override def op(): Unit = {
-    // TODO: The current implementation picks the number of buckets from session config.
-    //   This should be user-configurable to allow maintain the existing bucket numbers
-    //   in the index log entry.
+    // TODO: The current implementation picks the number of buckets INDEX_LINEAGE_ENABLED from
```

Comment:
What is INDEX_LINEAGE_ENABLED here?

Reply:
thanks @imback82 , cleaned up this message